src/bin/tikv-server.rs

```diff
@@ -89,8 +89,8 @@ fn get_store_path(matches: &Matches) -> String {
 
 fn run_local_server(listener: TcpListener, store: Storage) {
     let mut event_loop = create_event_loop().unwrap();
-    let trans = Arc::new(RwLock::new(MockTransport));
-    let mut svr = Server::new(&mut event_loop, listener, store, trans).unwrap();
+    let raft_router = Arc::new(RwLock::new(MockRaftStoreRouter));
```

Comment:
we use raft to emphasize this is a router for raft store. So I think it is ok. 

@ngaut